_ptbr/reply-one.md

```diff
@@ -21,9 +21,14 @@ Mamãe abriu caminho no meio da multidão e conseguimos lugares no ônibus. Ela
 
 Eu vomitei naquele ônibus.
 
-Eu me lembro de você dizendo que se juntaria a nós em breve e que enquanto isso, a gente deveria aproveitar Belgrado. Mas não estamos em Belgrado. Estamos em uma pequena vila do interior chamada Divljana, perto de Niš, em uma colônia de férias chamada "Sutjeska". Espero que você venha pra cá em breve.
+Eu me lembro de você dizendo que se juntaria a nós em breve e que enquanto isso, a gente deveria aproveitar Belgrado. Mas não estamos em Belgrado. Estamos em uma pequena vila do interior chamada Divljana, perto de Niš, em uma colônia de férias<sup aria-describedby="footnote-label" class="footnote-index"><a name="footnote-index-1" href="#footnote-1">1</a></sup> chamada "Sutjeska". Espero que você venha pra cá em breve.
 
 Com muito amor,
 Mamãe, Stane e eu.
 
 <div class="Response-signature">Kosta</div>
+
+<footer class="footnote">
```

Comment:
Can you just please use `Footer` and `Footer-index`, I know it looks a little bit weird in this stack, but the whole website uses this format. Thank you!